wp-content/themes/core/components/blocks/lead_form/css/lead-form.pcss

```diff
@@ -86,26 +82,20 @@
 
 .b-lead-form__form {
 	margin-top: var(--spacer-30);
+	max-width: var(--grid-5-col);
 
-	/* CASE: Lead Form: Grid Width */
-	.b-lead-form--width-grid & {
-		@media (--viewport-full) {
-			margin-top: var(--spacer-40);
-		}
+	@media (--viewport-full) {
+		width: calc(41.6667% - var(--grid-gutter-half));
```

Comment:
Minor, but it might be nice to add a short comment here mentioning the number of columns + gutter this equates to. I.e - something like `/* 5 of 12 columns + gutter */`